migration-engine/cli/src/commands.rs

```diff
@@ -44,9 +44,13 @@ impl Cli {
     }
 
     pub(crate) async fn run_inner(&self) -> Result<String, CliError> {
-        match self.command {
```

Comment:
Maybe you forgot this?

Reply:
indeed — I'm not comfortable with making it `Copy` however, in case we add a bunch of string params later. I'll make `run_inner` and `run` take `self` instead of `&self` instead.